platform/storage/zephyr/nvs/src/mender-storage.c

```diff
@@ -279,7 +280,7 @@ mender_storage_set_provides(mender_key_value_list_t *provides) {
     }
 
     /* Write provides */
-    if (nvs_write(&mender_storage_nvs_handle, MENDER_STORAGE_NVS_PROVIDES, provides_str, strlen(provides_str) + 1) < 0) {
+    if (nvs_write(&mender_storage_nvs_handle, filename, provides_str, strlen(provides_str) + 1) < 0) {
```

Comment:
`nvs_write()` takes a handle and then an integer, right? `filename` is a `char *`.